server/service/apple_mdm.go

```diff
@@ -1048,6 +1048,7 @@ var enrollmentProfileMobileconfigTemplate = template.Must(template.New("").Parse
 			<key>ServerCapabilities</key>
 			<array>
 				<string>com.apple.mdm.per-user-connections</string>
+				<string>com.apple.mdm.bootstraptoken</string>
```

Comment:
it's imperative that we add this to the enrollment profile, without a bootstrap token we can't remote wipe or lock

Reply:
@noahtalerman in regards to your question here: https://github.com/fleetdm/fleet/issues/9951#issuecomment-1440411070

this is how we get the bootstrap token